doc/nrf/libraries/networking/nrf_cloud_agps.rst

```diff
@@ -42,6 +42,15 @@ If :kconfig:`CONFIG_NRF_CLOUD_REST` is enabled, the :c:func:`nrf_cloud_rest_agps
 When nRF Cloud responds with the requested A-GPS data, the :c:func:`nrf_cloud_agps_process` function processes the received data.
 The function parses the data and passes it on to the modem.
 
+Optimizing cloud data downloads
+*******************************
+
+When the application only requires a fast GNSS fix at most once per 2 hour period, it can reduce LTE-M data charges by enabling :kconfig:`CONFIG_NRF_CLOUD_AGPS_FILTERED`.
```

Comment:
Better use 'LTE' to cover both M1 and NB-IoT.
```suggestion
When the application only requires a fast GNSS fix at most once per 2 hour period, it can reduce LTE data charges by enabling :kconfig:`CONFIG_NRF_CLOUD_AGPS_FILTERED`.
```